projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details/query-details.component.html

```diff
@@ -0,0 +1,7 @@
+<ng-container *ngIf="query">
+    <p>
+        <br><a class="govuk-link" href="javascript:void(0)" (click)="onBack()">Back to queries</a>
```

Comment:
@DavidJayakumar - `Back to queries` needs to run through `rpxTranslate` pipe for Welsh language support